runtime/tr.source/trj9/build/rules/ztpf-gcc/common.mk

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
Are the dates accurate here?

Reply:
There should only be one copyright date.